README.md

```diff
@@ -1,34 +1,70 @@
+
+The Contributor Guide provides detailed instruction on how to get your ideas and bug fixes seen and accepted, including:
 - [Code of Conduct](code-of-conduct.md)
 
-<h1>Contributing Guidelines: </h1>
+### How to file an issue 
+Not ready to contribute code, but see something that needs work? While the community encourages everyone to contribute code, it is also appreciated when someone reports an issue.
+Make sure to adhere to the prompted submission guidelines while opening an issue. 
+
+### How to find something to work on
+Kubesimplify is Fresh starting Community so there is a lot to do. Be in touch with [Discord Server](https://discord.gg/uFBC8MyeeU), Twitter and [official youtube Channel](https://www.youtube.com/c/saiyam911). There are a lot of ongoing projects and you can contribute to them.  
+### How to open a pull request
+We recommend open an issue before editing and Pulling request. If you don't know "How to open a Pull request" We refer you to [this resource](https://opensource.com/article/19/7/create-pull-request-github)
 
-Kubesimplfy's new website. This codebase is what runs on [Kubesimplify](https://kubesimplify.github.io/website/). 
 
-For new developers getting started, here are some instructions to follow:
-Fork the repo by clicking the fork button. This will clone the repo in your account, changes done here will be shown in the main repo. 
- 
+## Prerequisites
+To use this repository and to Run Kubesimplify's website on your computer, you need the installed dependencies locally:
+
+Before you start, install the dependencies. Clone the repository and navigate to the directory:
 Clone the repo to your local device: 
+
 ```bash
    git clone https://kubesimplify.github.io/website/ 
    cd website
 ```
-Install dependencies:
+Install [npm](https://www.npmjs.com/)
+
 ```bash
     $ npm install 
 ```
-To run the website on your computer:
+
 ```bash
     $ npm start
 ```
+
 And then open up your web browser and go to [localhost:3000](http://localhost:3000).
 
-## <h1>Code of conduct</h1>
-Participation in the Kubesimplify community is governed by the [CNCF Code of Conduct](https://github.com/cncf/foundation/blob/master/code-of-conduct.md).
+## Find something to work on
+
+The first step to getting starting contributing to Kubesimplify is to find something to work on. Help is always welcome, and no contribution is too small!
+Become a part of Kubesimplify community by Contribute. All types of contribution are allowed (code contribution, Docs contribution and community part too)
+
+Here are some things you can do today to get started contributing:
+
+1. Help improve the Kubesimplify documentation
+2. Clarify code, variables, or functions that can be renamed or commented on
+3. Write test coverage
+4. Help triage issues
+
+## Community Code of Conduct
+We follow Code of conduct here. Participation in the Kubesimplify community is governed by the [CNCF Code of Conduct](https://github.com/cncf/foundation/blob/master/code-of-conduct.md)
```

Comment:
Should this be CNCF code of conduct @saiyam1814 @anubha-v-ardhan

Reply:
@AvineshTripathi originally It was added as CNCF code of conduct.